mpv.py

```diff
@@ -35,8 +35,19 @@
 import re
 import traceback
 
+backend = None
+if "MPV_DYLIB_PATH" in os.environ:
+    backend = CDLL(os.environ["MPV_DYLIB_PATH"])
```

Comment:
在 *nix 系统上，加载这个前似乎似乎设置 locale，把这个提前的话，感觉有一些风险？

Reply:
请问是什么风险?没看出来.
没有在*nix系统上遇到过locale问题.

Reply:
它在注释里面写的：

> libmpv requires LC_NUMERIC to be set to "C". Since messing with global variables everyone else relies upon is
> still better than segfaulting, we are setting LC_NUMERIC to "C".

Reply:
已修改